packages/bootstrap-vue-next/src/plugins/popoverController/index.ts

```
@@ -1,134 +1,204 @@
+          const item = store.value.get(_id)
+          if (!item) return
+          item.promise.stop?.()
+          if (item.modelValue) {
+            await new Promise(async (resolve) => {
+              item.modelValue = false
+              item['onHidden'] = () => {
+                resolve(undefined)
+              }
+              await nextTick()
+              item['onUpdate:modelValue']?.(false)
+            })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)